virttest/utils_libvirt/libvirt_disk.py

```diff
@@ -510,3 +510,30 @@ def check_in_vm(vm, target, old_parts, is_equal=False):
     except Exception as e:
         logging.error(str(e))
         return False
+
+
+def create_remote_disk_by_first(vm, params):
+    """
+    Create an empty image on remote host
```

Comment:
How about making the name and the docstring more specific?
This function is to create a disk with the same name, size, type and path as the VM's first disk on the remote host. It seems that the current ones are too simple.. What do you think?

Reply:
Change to create_remote_disk_by_same_metadata()